### PR TITLE
Add wire loading delay to the button

### PR DESCRIPTION
**Pull request by putera**
❤️  the great project. I would love to contribute other component as well. teach me how if the component depent on external css and scripts. thanks.

### CODE REVIEW

**Comment by ph7jack** [comment]
Hello @putera
Thanks for your contribution
For information about the contribution, see https://livewire-wireui.com/docs/contributing

File: resources/views/components/button.blade.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class="w-4 h-4"
             @if (preg_replace('/[^a-zA-Z]+/', '', $spinner))
                 wire:target="{{ $spinner }}"
             @endif
-            wire:loading>
+            wire:loading.delay>
```
Comment (ph7jack) [inline resources/views/components/button.blade.php]: I would like to pass the delay modifier, e.g.
`<x-button ... loading-delay="shortest">`

```suggestion
            wire:loading.delay{{ $loadingDelay ? ".{$loadingDelay}":'' }}>
```

Like these examples, but you are free to use any way

**Review by ph7jack — CHANGES_REQUESTED** [review]
I like this idea, and thanks
Can you implement the loading delay modifier?

File: resources/views/components/button.blade.php (hunk 1 above)
Comment (putera) [reply resources/views/components/button.blade.php]: Yes. I have added the loadingDelay to the BaseButton file 👍🏽

**Review by ph7jack — APPROVED** [review]
Thanks :)

**Pull request merged**